modules/ipi-modify-a-disconnected-registry-config-yaml.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+
+// * installing-mirroring-creating-registry.adoc
+
+:_content-type: PROCEDURE
+[id="ipi-modify-a-disconnected-registry-config-yaml_{context}"]
+= Modify the `install-config.yaml` file to use the disconnected registry
+
+The `install-config.yaml` file must contain the disconnected registry node's certificate and registry information.
+
+.Procedure
+
+. Edit the `install-config.yaml` file to provide the mirror information for the registry.
+.. Update the `pullSecret` value to contain the authentication information from your registry:
++
+[source,terminal]
+----
+$ pullSecret: '{"auths":{"<mirror_host_name>:5000": {"auth": "<credentials>","email": "you@example.com"}}}'----
+----
++
+For `<mirror_host_name>`, specify the registry domain name that you specified in the certificate for your mirror registry, and for `<credentials>``, specify the base64-encoded user name and password for your mirror registry.
+
+.. Add the `additionalTrustBundle` parameter and value:
++
+[source,yaml]
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+  -----END CERTIFICATE-----
+----
++
+The value must be the contents of the certificate file that you used for your mirror registry, which can be an existing, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
```

Comment:
Vale flagged this as 33 words. I would break it up a bit. Maybe end the first sentence after "your mirror registry." Then,

"This certificate file can be an existing, trust..."